### ISSUE TITLE
[bug] Blank screen on opening or creating any GraphQL

### ISSUE THREAD

**Issue by lujop** [usebruno/bruno#867]
After updating to Bruno 1.0 when I open or create a GraphQL Post request I got a blank screen.
In console, I see this error:
```
framework-49f1e091cbf6b261.js:formatted:4188 Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```
I can reproduce just creating a new Request of type GraphqL

Info: 
```
Bruno 1.0.0 installed in MacOsX 14.0 installed by brew
electron | : 21.1.1
chrome | : 106.0.5249.103
node | : 16.16.0
v8 | : 10.6.194.17-electron.0
```

**Comment by helloanoop** [comment]
This is fixed in 1.0.1 @lujop

**Comment by lujop** [comment]
Thanks, I've just updated it to 1.0.1, and now it works correctly.
